daemon/inspect.go

```diff
@@ -151,3 +166,30 @@ func (daemon *Daemon) VolumeInspect(name string) (*types.Volume, error) {
 	}
 	return volumeToAPIType(v), nil
 }
+
+func getBackwardsCompatibleNetworkSettings(settings *network.Settings) *v1p20.NetworkSettings {
```

Comment:
@tiborvaas Looks like for the client using v1.21 they don't even see these old fields which means they are deprecated for v1.21. For older clients they will be empty if somebody used a non-default network but those older clients cannot even create a non-default network. So I am good with this. Thanks for the clarification.